probe-rs/examples/xtensa.rs

```diff
@@ -0,0 +1,47 @@
+use anyhow::Result;
+use probe_rs::config::ScanChainElement;
+use probe_rs::{Lister, MemoryInterface, Probe};
+
+fn main() -> Result<()> {
```

Comment:
Doesn't mean we cannot start writing better code than I or others wrote =)

Reply:
This example is unlikely to stay, at least in its current form. Currently, because we don't have a way to create an xtensa `Session`, we're using this example to test everything. Eventually, it will be phased out.